README.md

```diff
@@ -1,6 +1,6 @@
 # Glutil
 
-A collection of utilities for managing AWS Glue partitions.
+A collection of utilities for managing partitions of tables in the AWS Glue Data Catalog that are built on datasets stored in S3.
 
 ## Hello Friends
```

Comment:
Can we delete this section/modify to a reporting issues section since `glutil` is now open-sourced?